apps/vpnmanager/src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,32 @@
+import NextAuth from "next-auth";
+import GoogleProvider from "next-auth/providers/google";
+
+export default NextAuth({
+  secret: process.env.SECRET,
+  debug: true,
+  providers: [
+    GoogleProvider({
+      clientId: process.env.NEXT_APP_GOOGLE_CLIENT_ID ?? "",
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET ?? "",
+      authorization: {
+        params: {
+          scope: "openid email profile",
+        },
+      },
+    }),
+  ],
+  pages: {
+    signIn: "/login",
+    error: "/login",
+  },
+  callbacks: {
+    async signIn({ profile }) {
+      const allowedEmails = (process.env.ALLOWED_EMAILS ?? "").split(",");
```

Comment:
I could be asking for too much, but I think it would be better if these allowed emails were coming from a database (since we already have SQLite) instead of the .env file.

Reply:
@m453h your proposal makes sense. However I made this decision based on
- The list of people going to access the app is not going to change frequently, it will probably be a small number.
- We do not intend to have a UI to manage people who can access.

Reply:
Cool, I got you...based on the case it seems to be the quickest solution 👍